streams/src/test/java/org/apache/kafka/streams/integration/VersionedKeyValueStoreIntegrationTest.java

```diff
@@ -361,6 +365,91 @@ public void shouldCreateGlobalTable() throws Exception {
         }
     }
 
+    @Test
+    public void shouldManualUpgradeFromNonVersionedToVersioned() throws Exception {
+        // build non-versioned topology and start app
+        StreamsBuilder streamsBuilder = new StreamsBuilder();
+
+        streamsBuilder
+            .addStateStore(
+                Stores.timestampedKeyValueStoreBuilder(
```

Comment:
The docs (pre PR) might also be affected, but we do support upgrade even from plain KV-stores? If yes, we should test it, too, if no, we should be explicit in the docs.

Reply:
Yeah, as long as the changelog topic format is the same it'll work, which means non-timestamped stores are eligible for upgrade too. Added an additional test.

The docs PR just says "non-versioned" right now (no mention of timestamped vs non-timestamped). I can add a line to make it clear that upgrading from both timestamped and non-timestamped stores are supported.

Reply:
> I can add a line to make it clear that upgrading from both timestamped and non-timestamped stores are supported.

Sounds like a good idea. Thx.